internal/config/defaults.go

```diff
@@ -0,0 +1,7 @@
+package config
+
+func (c *Config) AddDefaults() {
```

Comment:
GoDoc missing

Reply:
If you need GoDoc here, the package or function name needs work.

I thought config.AddDefaults is obvious.